packages/gafl-webapp-service/src/processors/concession-helper.js

```diff
@@ -0,0 +1,105 @@
+import { CONCESSION, CONCESSION_PROOF } from './mapping-constants.js'
+/*
+ * I don't normally like this kind of code but this provides a single point at which a change can be made to
+ * move to a situation where multiple concessions are enabled
+ */
+const clear = licensee => {
+  delete licensee.concessions
+}
+
+const addJunior = licensee => {
+  if (!hasJunior(licensee)) {
+    if (hasSenior(licensee)) {
+      removeSenior(licensee)
+    }
+    if (hasDisabled(licensee)) {
+      removeDisabled(licensee)
+    }
+    if (!licensee.concessions) {
+      licensee.concessions = []
+    }
+    licensee.concessions.push({
+      type: CONCESSION.JUNIOR,
+      proof: {
+        type: CONCESSION_PROOF.none
+      }
+    })
+  }
+}
+
+const hasJunior = licensee => licensee.concessions && licensee.concessions.find(c => c.type === CONCESSION.JUNIOR)
+
+const removeJunior = licensee => {
+  if (hasJunior(licensee)) {
+    licensee.concessions.splice(
```

Comment:
More efficient to do `licensee.concessions = licencee.concessions.filter(c => c.type !== CONCESSION_JUNIOR)`